### ISSUE TITLE
mscorlib.dll not found

### ISSUE THREAD

**Issue by cbare** [abock/conservatorio#3]
I got the following error from the console version:

```
$ ./conservatorio 
The assembly mscorlib.dll was not found or could not be loaded.
It should have been installed in the `/Library/Frameworks/Mono.framework/Versions/4.2.1/lib/mono/4.5/mscorlib.dll' directory.
```

The GUI version worked for me. I now have 127.7MB of JSON for my 5+ years of Rdio. Thanks, and nice work BTW.

**Comment by abock** [comment]
Thanks for the report. Looks like that might be an issue with Mono's mkbundle, which is what I was using to produce the fully contained command line client for Mac.

You can download the "Linux/Windows" version which is just the raw managed assembly and invoke it via your system Mono.

I'll look into the mkbundle issue.

Glad your data is safe!

**Comment by abock** [comment]
Fixed in 1.2: http://conservator.io

Thanks!
